### DISCUSSION TITLE
dashboard/projects/28JOQgkiSHMc4hTy984SaUYdxmD3/x8GHsxYQ8KywgRwvkUSu

### DISCUSSION THREAD

**Discussion by giscus** [Meshojs/lloid-comment#2] [General]
# dashboard/projects/28JOQgkiSHMc4hTy984SaUYdxmD3/x8GHsxYQ8KywgRwvkUSu

Generated by Next.js

http://localhost:3000/dashboard/projects/28JOQgkiSHMc4hTy984SaUYdxmD3/x8GHsxYQ8KywgRwvkUSu

**Comment by Meshojs** [comment]
I really liked how simple was this documentation.
Can you explain this for me ?

$$ Loss=N1​i=1∑N​(yi​−y^​i​)2 $$

**Reply by Meshojs** [reply]
Yea sure! to measure the loss function!
you have to subtract the `Labels/target` from `Y_pred`
 * called MSE.
[source](https://docs.pytorch.org/docs/stable/generated/torch.nn.MSELoss.html)

**Reply by Meshojs** [reply]
# you can also comment here !

**Comment by Meshojs** [comment]
This was really helpful! thank you!

**Reply by Meshojs** [reply]
Yup

**Comment by Meshojs** [comment]
What was the in deep learning !

**Comment by Meshojs** [comment]
Thank you for this !
